### PR TITLE
Issue #2375: Add a view for managing files.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/2375

### CODE REVIEW

**Comment by mikemccaffrey** [comment]
Hmm... why am I not seeing a view for the files in the testing site?

**Comment by jenlampton** [comment]
Are there any files? Maybe we should add some empty text.

**Comment by quicksketch** [comment]
Yeah the view itself seems to be missing. There's no additional default config file in the diff. Perhaps it was left out of the PR?

**Comment by jenlampton** [comment]
> Yeah the view itself seems to be missing. 

whaaa!!!! sorry about that. Added back. I also moved all the views file stuff to file module in this PR since it seemed weird to be adding new things into system. I left file entity over there tho. We can move that in https://github.com/backdrop/backdrop-issues/issues/213

File: core/modules/file/config/views.view.file_admin.json [hunk 1]
```diff
@@ -0,0 +1,467 @@
+{
+    "_config_name": "views.view.file_admin",
+    "name": "file_admin",
+    "description": "",
+    "tag": "default",
+    "disabled": false,
+    "base_table": "file_managed",
+    "human_name": "Administer files",
+    "core": "1.5.x-dev",
```
Comment (klonos) [inline core/modules/file/config/views.view.file_admin.json]: Is this core version declaration required? What purpose does it serve?

Comment (jenlampton) [reply core/modules/file/config/views.view.file_admin.json]: I don't know what that does, it was added by views!

Comment (jenlampton) [reply core/modules/file/config/views.view.file_admin.json]: removed :)

Comment (klonos) [reply core/modules/file/config/views.view.file_admin.json]: https://github.com/backdrop/backdrop-issues/issues/2432 😉

**Comment by quicksketch** [comment]
I created a new PR replacing this one at https://github.com/backdrop/backdrop/pull/1696.

**Pull request closed without merging**